### DISCUSSION TITLE
Cmdlet to download asset from shared asset library

### DISCUSSION THREAD

**Discussion by FH-Inway** [d365collaborative/d365fo.tools#680] [Ideas]
@ Oleksandr Nikolaiev describes in https://www.yammer.com/dynamicsaxfeedbackprograms/#/Threads/show?threadId=1976607079530496 how downloading an asset from the shared asset library can be automated. Looks to me like this is similar to other LCS actions that the d365fo.tools already support.

Though not part of the official LCS API and arguably an incorrect behavior that Microsoft may or may not fix in the future, I think it would be worthwhile to add a cmdlet for it.

**Comment by Splaxi** [comment]
Agreed

**Comment by FH-Inway** [comment]
This is now available via the [Get-D365LcsSharedAssetFile](https://github.com/d365collaborative/d365fo.tools/wiki/Get-D365LcsSharedAssetFile) cmdlet. See also #687
